src/services/task.list.service.ts

```diff
@@ -26,7 +24,7 @@ export const initTaskList = (companyNumber: string,
   const allTasks = {
     officers: {
       state: toTaskState(csSubmission.data?.activeOfficerDetailsData?.sectionStatus),
-      url: urlUtils.getUrlWithCompanyNumberTransactionIdAndSubmissionId(officerSection(), companyNumber, transactionId, submissionId)
+      url: urlUtils.getUrlWithCompanyNumberTransactionIdAndSubmissionId(ACTIVE_OFFICERS_DETAILS_PATH, companyNumber, transactionId, submissionId)
```

Comment:
i think we still need the feature flag here as this looks like it will always go to the 'new' officers details path. 
maybe have the officerSection function back in but have it return ACTIVE_OFFICERS_DETAILS_PATH instead of NATURAL_PERSON_SECRETARIES_PATH like
`const officerSection = (): string => {
  if (FEATURE_FLAG_FIVE_OR_LESS_OFFICERS_JOURNEY_21102021 === 'true') {
    return ACTIVE_OFFICERS_DETAILS_PATH;
  } else {
    return ACTIVE_OFFICERS_PATH;
  }
};`